workspace_integration_test.go

```diff
@@ -2747,7 +2748,7 @@ func TestWorkspacesAutoDestroy(t *testing.T) {
 	})
 	t.Cleanup(wCleanup)
 
-	require.Equal(t, wTest.AutoDestroyAt, autoDestroyAt)
+	require.Equal(t, autoDestroyAt, wTest.AutoDestroyAt)
```

Comment:
swapped the order on these since require wants `require.Equal(t, <expected>, <actual>)`